selfservice/strategy/saml/strategy/test/strategy_helper_test.go

```diff
@@ -0,0 +1,154 @@
+package saml_test
+
+import (
+	"context"
+	"fmt"
+	"github.com/ory/dockertest/v3"
+	"github.com/ory/dockertest/v3/docker"
+	"github.com/ory/kratos/driver"
+	"github.com/ory/kratos/driver/config"
+	"github.com/ory/kratos/identity"
+	"github.com/ory/kratos/selfservice/strategy/saml"
+	"github.com/ory/x/logrusx"
+	"github.com/ory/x/resilience"
+	"github.com/phayes/freeport"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/net/html"
+	"net/http"
+	"net/http/cookiejar"
+	"net/http/httptest"
+	"testing"
+	"time"
```

Comment:
Can you please order the imports this way in all files ? 

```
import (
   "std libs"

  " project libs"

"external dependencies"
)
```

In the case of this file, that would be 

```suggestion
	"context"
	"fmt"
	"net/http"
	"net/http/cookiejar"
	"net/http/httptest"
	"testing"
	"time"
	
	"github.com/ory/dockertest/v3"
	"github.com/ory/dockertest/v3/docker"
	"github.com/ory/kratos/driver"
	"github.com/ory/kratos/driver/config"
	"github.com/ory/kratos/identity"
	"github.com/ory/kratos/selfservice/strategy/saml"
	"github.com/ory/x/logrusx"
	"github.com/ory/x/resilience"
	
	"github.com/phayes/freeport"
	"github.com/pkg/errors"
	"github.com/stretchr/testify/require"
	"golang.org/x/net/html"
```

Reply:
Have run `make format`